pages/jhipster_uml.md

```diff
@@ -112,7 +112,7 @@ JHipster UML is a tool that _can_ be used with the JHipster's generator. If you'
 ***
 
 # <a name="howtouse"></a>How to use it
-JHipster-UML is quite easy to use, you only need a class diagram exported in XMI and JHipster-UML will parse it to create your entities.
+JHipster-UML is not hard to use, you only need a class diagram exported in XMI and JHipster-UML will parse it to create your entities.
```

Comment:
Is there another way to say it without using hard ? Using intuitive for instance ? Or maybe just remove the first part of the sentence ?

Reply:
In fact this was one of tricky parts. Will update it. Thanks